whole-tale/ngx-dashboard
# Upgrade to Angular 9

## Problem
Angular 7 is already a bit stale.

Fixes #57 

## Approach
* Migrate from Angular 7 to Angular 8
* Migrate from Angular 8 to Angular 9
* Strip out many extraneous / unused dependencies (`@universal` `@ngx-cache`, `@ngx-meta`, `@ngx-config`, `@ngx-translate`, etc)
* Merged `@framework/` into `@shared/`
* Fixed line-endings / build warnings throughout

NOTE: this is an enormous PR that will likely have several potentially complex merge conflicts

## How to Test
1. Checkout this branch locally, rebuild the dashboard
2. Login to the WholeTale Dashboard
3. Run a basic smoke test
    * You should see that the platform behaves the same as previously
